### PR TITLE
Qt: Make network disabled icon larger and 50% opaque

### PR DESCRIPTION
**Pull request by luke-jr**
Clean alternative to #9145

### CODE REVIEW

**Comment by jonasschnelli** [comment]
Your PNG is 10bytes larger... :-)
```
-rw-r--r--  1 jonasschnelli  staff  2444 13 Nov 20:24 network_disabled_luke.png
-rw-r--r--  1 jonasschnelli  staff  2432 13 Nov 20:25 network_disabled_marco.png
```

I slightly prefer this solution over #9145.

**Comment by paveljanik** [comment]
ACK https://github.com/bitcoin/bitcoin/pull/9150/commits/990c56a1895508ee0822a1f7ac6c4be13163d97b

File: src/qt/res/src/network_disabled.svg [hunk 1]
```diff
@@ -38,30 +38,30 @@
...
+             style="color:#000000;font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:medium;line-height:normal;font-family:sans-serif;text-indent:0;text-align:start;text-decoration:none;text-decoration-line:none;text-decoration-style:solid;text-decoration-color:#000000;letter-spacing:normal;word-spacing:normal;text-transform:none;direction:ltr;block-progression:tb;writing-mode:lr-tb;baseline-shift:baseline;text-anchor:start;white-space:normal;clip-rule:nonzero;display:inline;overflow:visible;visibility:visible;opacity:1;isolation:auto;mix-blend-mode:normal;color-interpolation:sRGB;color-interpolation-filters:linearRGB;solid-color:#000000;solid-opacity:1;fill:#000000;fill-opacity:0.5;fill-rule:nonzero;stroke:none;stroke-width:1;stroke-linecap:butt;stroke-linejoin:miter;stroke-miterlimit:4;stroke-dasharray:none;stroke-dashoffset:0;stroke-opacity:1;color-rendering:auto;image-rendering:auto;shape-rendering:auto;text-rendering:auto;enable-background:accumulate" />
```
Comment (MarcoFalke) [inline src/qt/res/src/network_disabled.svg]: The issue is that those paths are overlapping, so I don't see another way than to merge them...

**Comment by luke-jr** [comment]
#9145 looks better now, and @MarcoFalke has a point with the overlapping paths (although it's barely visible), so closing this

**Pull request closed without merging**